content/blogs.html.haml

```diff
@@ -25,8 +25,17 @@
     = partial('footer.html.haml')
 
     %div#github
-      %a{:href => "https://github.com/rtyler/jenkins.io/edit/master/content/#{page.relative_source_path}",
+      %a{:href => "https://github.com/jenkins-infra/jenkins.io/edit/master/content/#{page.relative_source_path}",
```

Comment:
Yes, but this template (`blogs.html.haml`) is only used in the beta site and will likely be deleted before we go live.